llvm/test/Feature/Repo/repo_prunning_dependents.ll

```diff
@@ -44,4 +44,4 @@ entry:
 
 !repo.tickets = !{!0}
 
-!0 = !TicketNode(name: "A", digest: [16 x i8]  c"\82k\8AM\E0$d\05v\B1\AC\D8\ACdG\DB", linkage: external, visibility: default, pruned: false)
+!0 = !TicketNode(name: "A", digest: [16 x i8]  c"#\a0PM*\03\c2\18\b4s\e0\03]ee\e0", linkage: external, visibility: default, pruned: false)
```

Comment:
What caused this hash to change?

Reply:
The digest of a global object accumulated the hash of the running passes. Since a pass (named “Dead Global Elimination”) is added after the RepoPruning pass, the hash of all running passes is changed. Therefore, the GO’s digest is changed.